test/Feature/HLSLLib/PartiallyMappedResources.test

```diff
@@ -0,0 +1,161 @@
+#--- source.hlsl
+
+StructuredBuffer<int4> X : register(t0);
+StructuredBuffer<int4> Y : register(t1);
+
+RWStructuredBuffer<int> Out : register(u2);
+RWStructuredBuffer<bool> CAFM : register(u3);
+
+[numthreads(1,1,1)]
+void main() {
+  // 4096 int4's inside X or Y occupy 64KB of data. 
+  // (4096 int4's * 4 ints * 4 bytes per int)
+  // So, any index into the buffer >= [4096] will access a new "tile"
+
+  uint idx = 0;
+
+  uint status;
+  int4 Result = X.Load(0, status);
+  bool CAFMResult = CheckAccessFullyMapped(status);
+  CAFM[idx] = CAFMResult;
+  if (CAFMResult)
+    Out[idx] = Result.x;
+  else {
+    if (all(Result == int4(0,0,0,0)))

```

Comment:
Is Result even defined if `CheckAccessFullyMapped` returns false? We've read from an unmapped tile in that case...